projects/plugins/jetpack/modules/sharedaddy/admin-sharing.css

```diff
@@ -450,4 +450,18 @@ body.settings_page_sharing .advanced input[type=submit] {
 
 .sharing-block-message__buttons-wrapper .button {
 	margin-right:1rem;
-}
\ No newline at end of file
+}
+
+
+.admin-sharing-settings__block-theme-description {
+	margin-top: 1rem;
+	font-style: italic;
+}
+
+a.dops-card__link {
```

Comment:
Is this necessary? Value is `pointer` by default for links.

Reply:
I think it wasn't for some reason here.